andela/ah-frontend-thanos
# #161348750  Setup SASS processing on the client

#### What does this PR do?
 - Setup SASS processing on the client

#### Description of Task to be completed?
 - Add node-sass
 - Add sass-loader to translate sass into css

#### How should this be manually tested?
 - Install dependencies using `npm install`
 - Running the app with `npm start` should display a `React here` message that was styled using a sass file.

#### What are the relevant pivotal tracker stories?
[#161348750](https://www.pivotaltracker.com/story/show/161348750)
